_includes/selector.html

```diff
@@ -631,9 +641,8 @@
             },
             getpipNotes() {
                 var notes = [];
-                notes = [...notes, "<code>cuDF</code>, <code>cuML</code>, and <code>cuGraph</code> pip packages are hosted on the NVIDIA Index<br>",
-                         'pip installation supports Python <code>3.8</code>, <code>3.9</code>, and <code>3.10</code><br>',
-                         'pip installation supports CUDA <code>11.2</code> - <code>11.8</code>'];
+                notes = [...notes, "<code>cuDF</code>, <code>cuML</code>, <code>cuGraph</code>, and <code>cuSpatial</code> pip packages are hosted on the NVIDIA Index<br>",
+                         'pip installation supports Python <code>3.8</code>, <code>3.9</code>, and <code>3.10</code><br>'];
```

Comment:
We dropped Python 3.8 in RAPIDS 23.06. https://docs.rapids.ai/notices/rsn0029/
```suggestion
                         'pip installation supports Python <code>3.9</code> and <code>3.10</code><br>'];
```

Reply:
True, was thinking that saying pip supports 3.9 and 10 is a little blocking and might make users that are locked to 3.8 for whatever reason not realize they can use the wheels - however, it's not officially supported so maybe overthinking here.

Reply:
Ah I misunderstood - ty for the clarity!

Reply:
Yup, of course!